Composer/packages/electron-server/src/main.ts

```diff
@@ -150,7 +153,7 @@ async function loadServer() {
   log(`Server started at port: ${serverPort}`);
 }
 
-async function main() {
+async function main(show = false) {
```

Comment:
Nit: I know with the default value the boolean type is inferred, but it would be nice to specify the type of the show param here.